grammar.js

```diff
@@ -2016,6 +2032,7 @@ module.exports = grammar({
             $.inline_comment,
             $.inline_math,
             $.variable,
+            $.inline_link_target,
```

Comment:
Here, I included in the `attached_modifier` group. Should the `[...]` and `{...}` link syntax elements also go in here? In the scanner they are treated differently so I'm not even sure if `inline_link_target` should be in here..